source/backend/dal/Repositories/PropertyActivityRepository.cs

```diff
@@ -142,6 +160,29 @@ public bool TryDelete(long activityId)
             return deletedSuccessfully;
         }
 
+        /// <summary>
+        /// Delete an activity, and all property-activity relationships.
+        /// </summary>
+        /// <param name="activityId"></param>
+        /// <returns>Boolean of deletion sucess.</returns>
+        public bool TryDeleteFile(long activityId, long managementFileId)
```

Comment:
This method looks off. The name implies that the file is being deleted, but the implementation deletes the activity instead. Is it supposed to be a "Delete All activities and then the file"?

Reply:
Some context required here, but the general pattern of this file (and others) is just:
propertyActivityRepository.TryDelete();
what is this deleting? presumably a property activity. Following this pattern, propertyActivityRepository.TryDeleteFile() is trying to remove a property activity related to a file.

Perhaps TryDeleteByFile?